catalog-template.yaml

```
@@ -1,1346 +1,3107 @@
+  - name: rhacs-operator.v4.5.3
+    replaces: rhacs-operator.v4.5.2
+    skipRange: ">= 4.4.0 < 4.5.3"
+  - name: rhacs-operator.v4.5.4
+    replaces: rhacs-operator.v4.5.3
+    skipRange: ">= 4.4.0 < 4.5.4"
+  - name: rhacs-operator.v4.5.5
+    replaces: rhacs-operator.v4.5.4
+    skipRange: ">= 4.4.0 < 4.5.5"
+  - name: rhacs-operator.v4.5.6
+    replaces: rhacs-operator.v4.5.5
+    skipRange: ">= 4.4.0 < 4.5.6"
+  - name: rhacs-operator.v4.5.7
+    replaces: rhacs-operator.v4.5.6
+    skipRange: ">= 4.4.0 < 4.5.7"
+  - name: rhacs-operator.v4.5.8
+    replaces: rhacs-operator.v4.5.7
+    skipRange: ">= 4.4.0 < 4.5.8"
+  - name: rhacs-operator.v4.5.9
+    replaces: rhacs-operator.v4.5.8
+    skipRange: ">= 4.4.0 < 4.5.9"
+  - name: rhacs-operator.v4.6.0
+    replaces: rhacs-operator.v4.5.9
+    skipRange: ">= 4.5.0 < 4.6.0"
+  - name: rhacs-operator.v4.6.1
+    replaces: rhacs-operator.v4.6.0
+    skipRange: ">= 4.5.0 < 4.6.1"
+  - name: rhacs-operator.v4.6.2
+    replaces: rhacs-operator.v4.6.1
+    skipRange: ">= 4.5.0 < 4.6.2"
+  - name: rhacs-operator.v4.6.3
+    replaces: rhacs-operator.v4.6.2
+    skipRange: ">= 4.5.0 < 4.6.3"
+  - name: rhacs-operator.v4.6.4
+    replaces: rhacs-operator.v4.6.3
+    skipRange: ">= 4.5.0 < 4.6.4"
+  - name: rhacs-operator.v4.6.5
+    replaces: rhacs-operator.v4.6.4
+    skipRange: ">= 4.5.0 < 4.6.5"
+  - name: rhacs-operator.v4.6.6
+    replaces: rhacs-operator.v4.6.5
+    skipRange: ">= 4.5.0 < 4.6.6"
+  - name: rhacs-operator.v4.6.7
+    replaces: rhacs-operator.v4.6.6
+    skipRange: ">= 4.5.0 < 4.6.7"
+  - name: rhacs-operator.v4.6.8
+    replaces: rhacs-operator.v4.6.7
+    skipRange: ">= 4.5.0 < 4.6.8"
+  - name: rhacs-operator.v4.7.0
+    replaces: rhacs-operator.v4.6.8
+    skipRange: ">= 4.6.0 < 4.7.0"
+  - name: rhacs-operator.v4.7.1
+    replaces: rhacs-operator.v4.7.0
+    skipRange: ">= 4.6.0 < 4.7.1"
+  - name: rhacs-operator.v4.7.2
+    replaces: rhacs-operator.v4.7.1
+    skipRange: ">= 4.6.0 < 4.7.2"
+  - name: rhacs-operator.v4.7.3
+    replaces: rhacs-operator.v4.7.2
+    skipRange: ">= 4.6.0 < 4.7.3"
+  - name: rhacs-operator.v4.7.4
+    replaces: rhacs-operator.v4.7.3
+    skipRange: ">= 4.6.0 < 4.7.4"
+  - name: rhacs-operator.v4.7.5
+    replaces: rhacs-operator.v4.7.4
+    skipRange: ">= 4.6.0 < 4.7.5"
+  - name: rhacs-operator.v4.8.0
+    replaces: rhacs-operator.v4.7.5
+    skipRange: ">= 4.7.0 < 4.8.0"
+  - name: rhacs-operator.v4.8.1
+    replaces: rhacs-operator.v4.8.0
+    skipRange: ">= 4.7.0 < 4.8.1"
+- schema: olm.deprecations
+  package: rhacs-operator
+  entries:
+  - reference:
+      schema: olm.channel
+      name: latest
+    message: |
+      The `latest` channel is no longer supported.  Please switch to the `stable` channel.
+  - reference:
+      schema: olm.channel
+      name: rhacs-3.62
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
```

Suggestion:
```suggestion
      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported. Find supported versions in RHACS support policy document: https://access.redhat.com/support/policy/updates/rhacs
```
It'd be best to run it by the docs team in #forum-acs-docs.
It's best to present only this message and not the entire PR. See https://redhat-internal.slack.com/archives/CGDK78KRV/p1749630106473859